.github/workflows/verify_newproject_script.yml

```diff
@@ -39,11 +39,7 @@ jobs:
           sdk install kotlin $kotlinVersion
           sdk install kscript $kscriptVersion
           echo $PATH >> $GITHUB_PATH
-
-      - name: Verify generating new project from template-xml
-        working-directory: scripts
-        run: kscript new_project.kts package-name=co.myxmlproject.example app-name="My XML Project" template=xml
-
+        
       - name: Set up JDK 17
         uses: actions/setup-java@v3
         with:
```

Comment:
Shall we move this to be the first step and remove the `Set up JDK 11` step on lines 14-18?

Reply:
@ryan-conway you covered this issue from your PR already https://github.com/nimblehq/android-templates/pull/540, right?

Reply:
@luongvo yes, nice spot 😆